Travis run unittest  failed

when add PR , the Traivs report the following error message :
=== RUN   TestCalcParticipantPeers
--- FAIL: TestCalcParticipantPeers (0.00s)
	node_utils_test.go:210: peers(7, 2, 5, 3, 4, 3): [4 2 1], [5 1 2 6], [6 1 5]
please have a check

--- a/consensus/vbft/node_utils_test.go
+++ b/consensus/vbft/node_utils_test.go
@@ -206,7 +206,7 @@ func testCalcParticipantPeers(t *testing.T, n, c int) {
 	for _, p := range pc {
 		peers[p] = true
 	}
-	if len(peers) <= 2*c+1 {
+	if len(peers) < 2*c+1 {
 		t.Fatalf("peers(%d, %d, %d, %d, %d, %d): %v, %v, %v", n, c, len(peers), len(pp), len(pe), len(pc), pp, pe, pc)
 	}
 }

